streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/TaskAssignor.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.UUID;
-import org.apache.kafka.streams.processor.internals.assignment.AssignorConfiguration.AssignmentConfigs;
```

Comment:
oops -- glad you picked up this refactor, this would have been embarrassing to ship 😅